src/traces/scatter3d/calc.js

```diff
@@ -9,6 +9,7 @@
 'use strict';
 
 var arraysToCalcdata = require('../scatter/arrays_to_calcdata');
+var calcLineColorscale = require('../scatter/line_colorscale_calc');
 var calcMarkerColorscale = require('../scatter/marker_colorscale_calc');
```

Comment:
`calcMarkerColoscale` and `calcLineColorscale` could probably be merged, if I'm not mistaken.